app/src/main/res/values/settings_keys.xml

```diff
@@ -16,8 +16,6 @@
     <string name="use_external_video_player_key">use_external_video_player</string>
     <string name="use_external_audio_player_key">use_external_audio_player</string>
 
-    <string name="volume_gesture_control_key">volume_gesture_control</string>
-    <string name="brightness_gesture_control_key">brightness_gesture_control</string>
```

Comment:
Since you are replacing these two settings with two other settings, I think you should create a settings migration to make sure the previous preferences are still respected after an app update.

Reply:
Alright. I am sorry, I have never done something like that before. Could you give me a hint how to implement the settings migration? Thank you in advance!

Reply:
The instructions are in the [SettingsMigrations](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/java/org/schabi/newpipe/settings/SettingMigrations.java) file

Reply:
Thank you!